src/Alert/Alert.test.js

```diff
@@ -1,7 +1,21 @@
-import { Alert } from '.'
+import React from 'react'
```

Comment:
I am just getting started with jest and am using this to test that it can be run and see how it integrates with storybook.